src/grndb.c

```diff
@@ -184,6 +196,16 @@ main(int argc, char **argv)
     grn_default_logger_set_max_level(log_level);
   }
 
+  int log_flags = GRN_LOG_DEFAULT;
```

Comment:
Could you reduce scope by putting this into `if (log_flags_name)` block?

Reply:
We don't need to initialize this by `GRN_LOG_DEFAULT`. Because the default value isn't used.

Reply:
Fixed.